### PR TITLE
Mobile functionality 

### LINKED ISSUE #11: Add button to go back home from account page
We should let people go back home 🤔

### LINKED ISSUE #14: build mobile responsive version of website

### PR DESCRIPTION
**Pull request by bhaviksingh**
## Changes

Added functionality to make the website mobile friendly (no video, just chat) and some other tweaks.
- On Mobile sizes, we use defaultRoom for all rooms, displaying only chat. 
- On Mobile sizes, special layout is used to stack footer and content
- Tweaked domRing to render nicely on mobile.
- Added back button to account page 
- Make it so that the offline rooms can be clicked into, to set stage for issue #7 

## Context

Context is that previously the site was just trying to render the entire desktop page on mobile, which was very broken. Now, we have a special mobile view. 

## Testing

Not tested in app, will do after deploy. Tested in inspector.

https://github.com/user-attachments/assets/3fd0f569-7330-434f-9cbc-3a094173aae6

## Relevant Issues

- Closes #14 
- Closes #11

### CODE REVIEW

**Comment by bhaviksingh** [comment]
There are some issues. Not quite ready for merging yet. 

IMG_0285
IMG_0284

**Comment by bhaviksingh** [comment]
At this point it works decently well. there are still some issues
- Scroll for chat isn't working super well on mobile
- The useMediaQuery hook isn't consistent. For that reason, the video sometimes plays when you open the page.

Will try and improve these before merging.

**Comment by bhaviksingh** [comment]
Updated!  Looks good now :)

https://github.com/user-attachments/assets/bcff83e0-7316-44dc-9076-46627a827c5c

**Comment by kyeah** [comment]
Noticed a few things! Experienced this on my iphone 15 in both safari and chrome; also replicated this on my desktop with a "mobile-ish" width and short height.

IMG_038AA819D7B1-1

- bottom-right menu overlaps with the chat input on short displays
- fade doesn't seem to work when scrolled all the way to the bottom of the chat...
- and then when i scroll up, I get weird fade behavior in the middle of the chat 🤔

File: client/src/pages/index.tsx [hunk 1]
```diff
@@ -41,31 +34,47 @@ const Index: NextPage = () => {
...
-         <title>THING</title>
-       </Head>
-       <div className="fullBleed noOverflow">
-         <DomRing />
-         <div className="center:absolute highestLayer center-text">
-         <Link href="/about">
-         thing.tube
-         </Link>
-         <br/>
-           next stream: 10/12/24
```
Comment (kyeah) [inline client/src/pages/index.tsx:60]: I think we wanted to keep these copy edits from sarah :D

**Review by kyeah — APPROVED** [review]

**Pull request merged**